sycl/test-e2e/Config/kernel_from_file.cpp

```diff
@@ -14,7 +14,7 @@
 // RUN: %if linux %{ llvm-link -o=%t_app.bc %t.bc %t_compiler_wrappers.bc %t_asan.bc %} %else %{ llvm-link -o=%t_app.bc %t.bc %t_compiler_wrappers.bc %}
 // >> ---- translate to SPIR-V
 // RUN: llvm-spirv -o %t.spv %t_app.bc
-// RUN: %clangxx -Wno-error=ignored-attributes %sycl_include -DSYCL_DISABLE_FALLBACK_ASSERT %cxx_std_optionc++17 %include_option %t.h %s -o %t.out %sycl_options -fno-sycl-dead-args-optimization -Xclang -verify-ignore-unexpected=note,warning
```

Comment:
What do you mean "unused"? According to the comment at line 6, it's required for this test.

Reply:
I guess it's required for the device compilation because host part is compiled in non-sycl mode. `-fno-sycl-dead-args-optimization` is ignored in non-sycl mode. 
Did I get it right? If so, please, update the test comment and commit message.